posts/2021/10/numpy_benchmarking_blog.md

```diff
@@ -0,0 +1,61 @@
+<!--
+.. title: NumPy Benchmarking
+.. slug: numpy-benchmarking
+.. date: 2021-09-27 12:23:40 UTC-05:00
+.. author: Khushi Agrawal
+.. tags: NumPy, Pythran, Numba, Transonic
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+.. previewimage:
+-->
+
+In this blog post, I'll be talking about my journey in Quansight. I want to share all things I was involved in and accomplished. What issues I faced, and most importantly, what were awesome life hacks I learned during this period.
+
+First of all, I'd like to express my gratitude to the whole team for allowing me to be a part of such a great team. My work was majorly focused on providing performance benchmarks to NumPy in realistic situations. The target was to show the world that NumPy is efficient in handling quasi real-life situations too.
+
+<p align="center">
+      <img src = "../../images/2021/10/journey.jpeg" alt = "My Journey!">
+</p>
```

Comment:
I think this is a good suggestion! Reading through the blog post, it seems like all of these things are mentioned so the content isn't necessarily missing. If I do think about this in terms of reading order, though, I think it might better preserve the kind of preview nature of the image to add the following:

<img src = "/images/2021/10/journey.jpeg" alt = "A word cloud with themes, open-source projects and people mentioned throughout the blog post. Each is stylized using a different font, most of them calligraphical.">

As always, this is just a suggestion! I think what is there now is a clear description, just less linked to the rest of the post.